### PR TITLE
Add GELF Input Plugin #93

### PR DESCRIPTION
**Pull request by fbalicchia**
Hi @megastef,
please check this pr. I've use jshint to have the same standard.
Thanks again for previous refactor. I've appreciated

### CODE REVIEW

**Comment by fbalicchia** [comment]
Close PR cause at the moment mapping is wrong as indicated in  https://github.com/sematext/logagent-js/issues/94 I'll pass the whole message

File: lib/plugins/input/command.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 'use strict'
 var split = require('split2')
-var momenttz = require('moment-timezone')
```
Comment (megastef) [inline lib/plugins/input/command.js]: I assume momenttz was not used (but it will be soon, when we format dates in the command plugin). Ok for now.

**Comment by megastef** [comment]
We have precommit hooks in package.json - thank you for pointing me to this issues. 

A manual check can be done with 
```npm run precommit```

To install the git precommit-hook run
```
npm run  hook-install
```
We should write it down in a contribution guide ...

**Comment by megastef** [comment]
See https://github.com/sematext/logagent-js/commit/32df3561ec035cb1fda3b8040c7ad09c2b75af20 fixed existing issues.

File: lib/plugins/input/influxHttp.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/*jshint esversion: 6 */
```
Comment (megastef) [inline lib/plugins/input/influxHttp.js]: I don't think we need jshint version - why is it injected here?

File: lib/plugins/input/gelf.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var gelfserver = require('graygelf/server')
+var safeStringify = require('fast-safe-stringify')
+var consoleLogger = require('../../util/logger.js')
+
+
+function InputGELF (config, eventEmitter) {
+  this.config = config.configFile.input.gelf
+  this.config.port = config.configFile.input.gelf.port || 12100
```
Comment (megastef) [inline lib/plugins/input/gelf.js]: Note, we should not use `config.configFile.` - this property was only for some Logagent 1.0 modules, or for a special case where one plugin needs to know if another plugin is present.  Using config.configFile limits plugin to a single instance and forces user to use the name "gelf" to make the config working. 

Normally we should address plugin configuration properties directly via "config.property" e.g. "config.port". 
See e.g.  https://github.com/sematext/logagent-js/blob/master/lib/plugins/input/elasticsearchHttp.js#L8

File: lib/plugins/input/influxHttp.js (hunk 2 above)
Comment (fbalicchia) [reply lib/plugins/input/influxHttp.js]: Jshint inform me that 
line 6, col 1, 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
To skip this warning I added 
/*jshint esversion: 6 */ 
but probably the the right way is add
"esversion"     : 6, to .jshintrc file

File: lib/plugins/input/gelf.js (hunk 3 above)
Comment (fbalicchia) [reply lib/plugins/input/gelf.js]: ok

**Pull request closed without merging**